azure-pipelines.yml

```diff
@@ -1,6 +1,7 @@
 variables:
   npm_config_cache: $(Pipeline.Workspace)/.npm
   node_version: '12.4.0'
+  MONGODB_VERSION: '4.2.3'
```

Comment:
This is a temporary fix for missing tarball in CDN.

For now we run tests against mongo 4.2.3 since 4.2.4 isn't available yet. We'll want to remove this later.